### PR TITLE
Link to manual docs instead of wiki for SPNEGO authentication

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: _includes/manuals/1.4/1.2_release_notes.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ A new user_data template type has been added and images in Foreman can be config
 
 #### Web UI Kerberos support
 
-Foreman can now be configured under Apache HTTP with mod_auth_kerb to authenticate users using GSSAPI and Kerberos, allowing single sign on (SSO) in enterprise environments.  This has been tested with [FreeIPA project](http://www.freeipa.org) and more information on configuring it can be found on the [mod_auth_kerb wiki page](http://projects.theforeman.org/projects/foreman/wiki/Foreman_and_mod_auth_kerb).
+Foreman can now be configured under Apache HTTP with mod_auth_kerb to authenticate users using GSSAPI and Kerberos, allowing single sign on (SSO) in enterprise environments.  This has been tested with [FreeIPA project](http://www.freeipa.org) and more information on configuring it can be found in the [SPNEGO documentation](/index.html#5.7SPNEGOauthentication).
```
Comment (domcleal) [inline _includes/manuals/1.4/1.2_release_notes.md]: Does that link work?  I think the manual has a base href set, so you need the full path including /manuals/{{ page.version}} etc.

Comment (stbenjam) [inline _includes/manuals/1.4/1.2_release_notes.md]: Oh, right, sorry. Fixed.

Comment (stbenjam) [inline _includes/manuals/1.4/1.2_release_notes.md]: Also not {{page.version}} since it's specifically the 1.4 release notes.

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**